### PR TITLE
Fix typing.Literal converter

### PR DESCRIPTION
**Pull request by nitros12**
### Description
Make the typing.Literal converter work

### Checklist

- [x] This PR makes changes to the code.
    - [ ] The changes are breaking.
    - [ ] The changes implement a new feature.
    - [x] The changes fix an issue.
    - [ ] I updated the documentation to reflect these changes.
- [ ] This PR makes changes to the documentation.
    - [ ] The changes require a different version of sphinx.
    - [ ] I updated the configuration file to reflect these changes.

- [x] This PR has been tested. (somewhat more than before)

### CODE REVIEW

File: discord/ext/alternatives/literal_converter.py [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 
 _old_actual_conversion = Command._actual_conversion
 
-def _actual_conversion(self, ctx, converter, argument, param):
-    if converter is Literal:
+async def _actual_conversion(self, ctx, converter, argument, param):
+    try:
+        origin = converter.__origin__
+    except AttributeError:
+        origin = None
```
Comment (Gobot1234) [inline discord/ext/alternatives/literal_converter.py:11]: Couldn't this use `typing.get_origin` as its already 3.8+?

Comment (nitros12) [reply discord/ext/alternatives/literal_converter.py:11]: Is the lib 3.8+?

Comment (Gobot1234) [reply discord/ext/alternatives/literal_converter.py:11]: typing.Literal is 3.8+

Comment (nitros12) [reply discord/ext/alternatives/literal_converter.py]: nvm, changed to use it.

**Pull request merged**